src/bucket.ts

```diff
@@ -3951,6 +4051,25 @@ class Bucket extends ServiceObject {
   getId(): string {
     return this.id!;
   }
+
+  disableAutoRetryConditionallyIdempotent_(
+    coreOpts: any,
```

Comment:
Is `any` the best option here? Can it be object?

Reply:
Object won't work, I suppose I could mimic / clone the type from common if that is preferred?

Reply:
I think either that or disabling linting on this line. Object from common is what we've done in the past, so might be a good idea. It creates duplicate code though :( 

Your call on what you think is best.